### ISSUE TITLE
Incorrect comment on cifar10.py

### ISSUE THREAD

**Issue by a7744hsc** [tensorflow/tensorflow#5251]
Hi there,
On line 259 of cifar10.py, the comment is `# softmax, i.e. softmax(WX + b)`, but actually this layer only did a liner calculation(wx+b). So I believe this is a little bit misunderstanding for a newbie like me.

**Comment by drpngx** [comment]
Feel free to submit a pull request

**Comment by a7744hsc** [comment]
Thanks @drpngx , I've submitted a pull request based on what I know.

**Comment by drpngx** [comment]
Thanks!
